interface/notes_for_s3_interface_deployment.md

```diff
@@ -0,0 +1,18 @@
+# Notes for deploying interface single page app to AWS S3 bucket
+
+Download and configure aws command line interface 
+(http://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html)
+
+Set up users, passwords and permissions with AWS IAM
+(http://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html)
+
+Initialize S3 bucket using AWS console or something like terraform
+(https://www.terraform.io/)
+
+Upload bin folder to S3 bucket using aws-cli 'sync' command
+
+```shell
+# Inside the current folder (adage-server/interface), run
+aws s3 sync ./bin s3://<your s3 bucket name>
+
+```
```

Comment:
Yes I can see that - I talked to @dcgoss and @cgreene, and since this is the way that things are deployed in cognoma, I decided to go ahead and try it out. It worked, and I put some notes for that process in this markdown file to keep track of it, in case we decide to deploy this way. We definitely don't need to nail this down right now (at least for this pull request) - we can all keep talking about it and decide what the best strategy for deployment is.